lib/explorer/series.ex

```diff
@@ -1405,6 +1405,117 @@ defmodule Explorer.Series do
   @spec at_every(series :: Series.t(), every_n :: integer()) :: Series.t()
   def at_every(series, every_n), do: apply_series(series, :at_every, [every_n])
 
+  @doc """
+  Picks values based on an `Explorer.Query`.
+
+  The query is compiled and runs efficiently against the series.
+  The query must return a boolean expression or a list of boolean expressions.
+  When a list is returned, they are joined as `and` expressions.
+
+  > #### Notice {: .notice}
+  >
+  > This is a macro. You must `require Explorer.Series` before using it.
+
+  Besides element-wise series operations, you can also use window functions
+  and aggregations inside comparisons.
+
+  See `filter_with/2` for a callback version of this function without
+  `Explorer.Query`.
```

Comment:
```suggestion
  `Explorer.Query`. See `mask/2` if you want to filter values based on another series.
```

Reply:
Addressed by: [d17236d](https://github.com/elixir-explorer/explorer/pull/728/commits/d17236d7cda59be71e4b01b1869dbff089c11356)

I'm taking no more chances with the formatter :)